src/components/Conversation/ContentCard.tsx

```diff
@@ -0,0 +1,112 @@
+"use client";
+import { useState } from "react";
+import { ViewPoint } from "@/types/conversations.types";
+import {
+    HandThumbUpIcon,
+    HandThumbDownIcon,
+    ArrowUpCircleIcon,
+} from "@heroicons/react/24/solid";
+type ContentCardProps = {
+    viewpoint: ViewPoint;
+};
+export default function ContentCard({ viewpoint }: ContentCardProps) {
+    const [thumbsup, setThumbsup] = useState(false);
+    const [thumbsdown, setThumbsdown] = useState(false);
+    const [arrowup, setArrowup] = useState(false);
+    const [thumbsupNum, setThumbsupNum] = useState(viewpoint.thumbsup);
+    const [thumbsdownNum, setThumbsdownNum] = useState(viewpoint.thumbsdown);
+    const [arrowupNum, setArrowupNum] = useState(viewpoint.up);
+
+    function HandleThumbsup() {
```

Comment:
After you refactor the state above, you should be able to significantly reduce the boilerplate code below